### ISSUE TITLE
Watching VOD is impossible because audio/video is out of sync

### ISSUE THREAD

**Issue by M0nty99** [peak3d/inputstream.adaptive#1036]
### Describe the bug

When watching on demand content the audio and video is out of sync. Sometimes the video also freezes while the audio continues to playback. 

## Expected Behavior
Play a VOD with normal audio/video sync

## Actual Behavior
Audio out of sync

### To Reproduce

Replay a VOD
Check for audio

### Debuglog
The debuglog can be found here:

https://pastebin.com/R6cNjpVU

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

Operating system version/name: Librelec 10.0.2

Kodi version: 19.4.0

inputstream.adaptive version: v19.0.7.1

note: Once the issue is made we require you to update it with new information should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.

**Comment by CastagnaIT** [comment]
Your log have debug disabled, so enable Kodi debug logging, then reboot kodi and reproduce the problem, after get the new log

please make sure that Kodi player "Video" setting "sync playback to display" is disabled

**Comment by M0nty99** [comment]
Sorry, I enabled the debug mode and reproduced the problem.
The mentioned video setting is disabled indeed.

https://paste.kodi.tv/inazenayud.kodi

Thanks!

**Comment by CastagnaIT** [comment]
from the log i dont see nothing that explain this problem on InputStream
in the first log there is some stream stalled that can means unstable connection or less bandwidth than required

i can suggest to try use that addon on a different operative system and see what happens,
and also contact that VOD addon developer to know if it is only a your problem o a generalized problem of all users that use that addon

**Comment by M0nty99** [comment]
Thank you for your reply, 

I'll contact the add-on dev as well and hear what they might say. My first thought was connection as well, but I have 500mbit down and 100mbit up. And I've been looking at my firewall when this happens and there is nothing being blocked, or any other traffic going through it. It seems somehow related with the playback on the pi, but can't figure out what suddenly caused this

**Comment by CastagnaIT** [comment]
have you asked elsewhere?
i see also that you are using a RPI, i know that RPI's has performance problems that cause stuttering when the video is played in software decoding

you can try check if you are playing as software decoding by pressing key "o" while in playback
to open info window, should be written in "Video decoder" info

i remember if things are not changed over the time that RPI has a kernel? setting to reserve more memory to GPU that could help, but i cannot help more than this

**Comment by M0nty99** [comment]
Hi,

I have asked some friends who also have a rpi with kodi to try and they are not experiencing the same issue. I tried a new, fresh install on my rpi as well and experienced the same problem. So I'm thinking it is most likely a hardware issue with my rpi to be honest. It has worked fine before, and is still working fine with other with same setup. So I don't think the problem is with the software or the rpi itself, but something possible faulty on my rpi. I also tried a different port on my switch and a different ethernet cable. Still the same result

Something new meanwhile also manifested when locally streaming content from my sambashare, there are random short (1sec) black screens sometimes. So I believe my rpi has an issue. 

I appreciate you following up on this issue btw! 

Thanks!
